performance/include/traccc/performance/timing_info.hpp

```diff
@@ -15,10 +15,14 @@
 
 namespace traccc::performance {
 
+using timing_info_pair = std::pair<std::string, std::chrono::nanoseconds>;
+
 /// Struct for storing time measurements collected in timer class
 ///
 struct timing_info {
-    std::vector<std::pair<std::string, std::chrono::nanoseconds>> data;
+    std::vector<timing_info_pair> data;
+
+    double get_time(const std::string_view timer_name);
```

Comment:
You should return an `std::chrono` object here. That's the whole point of those types. To avoid having to always rely on documentation for knowing what unit a given value is expressed in.